src/__tests__/Search.test.js

```diff
@@ -0,0 +1,16 @@
+import { render, screen } from "@testing-library/react";
+import Search from "../components/Search";
+
+describe("search", () => {
+   const { asFragment } = render(<Search />)
+   
+   it("renders correctly", () => {
+      expect(asFragment()).toMatchSnapshot();
+   });
+
+   it("renders search button correctly", () => {
+      const searchButton = screen.getByRole("button");
+      expect(searchButton).toBeInTheDocument();
+   })
+   
+   });
```

Comment:
This isnt working for a couple of reasons. For the first test, you should put the render method on line 5 into the test creating the snapshot, and in the second test you need to render the component again before you can try and search for any elements to run assertions on. Put together, this should look like this:

```
describe("search", () => {
  it("renders correctly", () => {
    const { asFragment } = render(<Search />);

    expect(asFragment()).toMatchSnapshot();
  });

  it("renders search button correctly", () => {
    render(<Search />);
    const searchButton = screen.getByRole("button");

    expect(searchButton).toBeInTheDocument();
  });
});
```